### ISSUE TITLE
Does not install on Surface Pro X

### ISSUE THREAD

**Issue by nashokin** [keeferrourke/capitaine-cursors#49]
Installation fails when installing via inf on Surface Pro X. Please fix or suggest what to edit in inf to make it work. Thank you.

**Comment by keeferrourke** [comment]
I don't use Windows, and don't consider the Windows port to be supported. Sorry :(

This is first and foremost a cursor pack for Linux/BSD systems.

**Comment by abbaccadd0** [comment]
> Installation fails when installing via inf on Surface Pro X. Please fix or suggest what to edit in inf to make it work. Thank you.

I install these cursors via .inf on Windows 11 and Windows 10 21H1, it worked
so maybe your system is broken, try sfc /scannow to fix it.

**Comment by abbaccadd0** [comment]
> I don't use Windows, and don't consider the Windows port to be supported. Sorry :(
> 
> This is first and foremost a cursor pack for Linux/BSD systems.

I like your cursor very much, your work is really good
but I notice the Windows version is much older than Linux version, will them be update? or do it by myself?
I tried build.sh on Windows but it seems not work, is this shell only work on Linux?

**Comment by keeferrourke** [comment]
> I tried build.sh on Windows but it seems not work, is this shell only work on Linux?

Yes, the build script is only for Linux/BSD. It generates an xcursor theme, which does not work on Windows.

Windows uses their own cursor format. It is possible to generate a cursor in the format from these sources (I've done it manually before), but I'm not aware of any sort of standard tooling to do so.

If you're confident with windows scripting and windows cursor formats, I would welcome PRs to get it building on Windows!

**Comment by abbaccadd0** [comment]
> > I tried build.sh on Windows but it seems not work, is this shell only work on Linux?
> 
> Yes, the build script is only for Linux/BSD. It generates an xcursor theme, which does not work on Windows.
> 
> Windows uses their own cursor format. It is possible to generate a cursor in the format from these sources (I've done it manually before), but I'm not aware of any sort of standard tooling to do so.
> 
> If you're confident with windows scripting and windows cursor formats, I would welcome PRs to get it building on Windows!
[Capitain Cursors R4 for Windows.zip](https://github.com/keeferrourke/capitaine-cursors/files/7546365/Capitain.Cursors.R4.for.Windows.zip)
I don't know how to use scripts, so I manually made the cursors of the current version with a resolution of 128*128.
:-D
